### ISSUE TITLE
Added Shopping website using html and css

### ISSUE THREAD

**Issue by Chirag0407** [adarshpand3y/Hacktoberfest2022-DSA#443]
Added Shopping website using html and css

**Comment by SahilJin123** [comment]
I want to do this

**Comment by inbuilt-aura** [comment]
Hello sir, can you assign this to me. I think I can help you as i have experience with building shopping and ecommerce website.

**Comment by inbuilt-aura** [comment]
Yeah,sure but whom have you sent pull request,send it to me so that i can
do it
